src/condor_startd.V6/ResAttributes.cpp

```diff
@@ -485,8 +485,11 @@ MachAttributes::compute_for_policy()
 	{ // formerly IS_TIMEOUT(how_much) && IS_SHARED(how_much)
 		m_load = sysapi_load_avg();
 
+
 		sysapi_idle_time( &m_idle, &m_console_idle );
 
+		//dprintf(D_ALWAYS | D_BACKTRACE, "MachAttributes::compute_for_policy idle=%d\n", m_idle);
+
```

Comment:
Do we want to keep this `dprintf`?

Reply:
not sure,  prefer to leave it commented out for now.